lib/internal/streams/end-of-stream.js

```diff
@@ -38,6 +38,15 @@ const {
   willEmitClose: _willEmitClose,
 } = require('internal/streams/utils');
 
+const {
+  isBrandCheck,
+} = require('internal/webstreams/util');
+
+const isReadableStream =
```

Comment:
Can these be imported from https://github.com/nodejs/node/blob/5117c6c9e7418eecd6e73145a1f8339e18db323f/lib/internal/webstreams/readablestream.js#L3193 and https://github.com/nodejs/node/blob/5117c6c9e7418eecd6e73145a1f8339e18db323f/lib/internal/webstreams/writablestream.js#L1302?

Reply:
Tried this it broke the build itself with the following errors:
```
node:internal/worker/io:331
class ReadableWorkerStdio extends Readable {
                                  ^

TypeError: Class extends value undefined is not a constructor or null
    at node:internal/worker/io:331:35
    at BuiltinModule.compileForInternalLoader (node:internal/bootstrap/loaders:333:7)
    at requireBuiltin (node:internal/bootstrap/loaders:364:14)
    at node:internal/webstreams/readablestream:70:5
    at BuiltinModule.compileForInternalLoader (node:internal/bootstrap/loaders:333:7)
    at requireBuiltin (node:internal/bootstrap/loaders:364:14)
    at node:internal/streams/end-of-stream:43:5
    at BuiltinModule.compileForInternalLoader (node:internal/bootstrap/loaders:333:7)
    at requireBuiltin (node:internal/bootstrap/loaders:364:14)
    at node:internal/streams/operators:20:22

Node.js v20.0.0-pre
Skipping doc-only (no crypto)
node:internal/worker/io:331
class ReadableWorkerStdio extends Readable {
                                  ^

TypeError: Class extends value undefined is not a constructor or null
    at node:internal/worker/io:331:35
    at BuiltinModule.compileForInternalLoader (node:internal/bootstrap/loaders:333:7)
    at requireBuiltin (node:internal/bootstrap/loaders:364:14)
    at node:internal/webstreams/readablestream:70:5
    at BuiltinModule.compileForInternalLoader (node:internal/bootstrap/loaders:333:7)
    at requireBuiltin (node:internal/bootstrap/loaders:364:14)
    at node:internal/streams/end-of-stream:43:5
    at BuiltinModule.compileForInternalLoader (node:internal/bootstrap/loaders:333:7)
    at requireBuiltin (node:internal/bootstrap/loaders:364:14)
    at node:internal/streams/operators:20:22

Node.js v20.0.0-pre
Skipping lint-js-doc (no crypto)
node:internal/worker/io:331
class ReadableWorkerStdio extends Readable {
                                  ^

TypeError: Class extends value undefined is not a constructor or null
    at node:internal/worker/io:331:35
    at BuiltinModule.compileForInternalLoader (node:internal/bootstrap/loaders:333:7)
    at requireBuiltin (node:internal/bootstrap/loaders:364:14)
    at node:internal/webstreams/readablestream:70:5
    at BuiltinModule.compileForInternalLoader (node:internal/bootstrap/loaders:333:7)
    at requireBuiltin (node:internal/bootstrap/loaders:364:14)
    at node:internal/streams/end-of-stream:43:5
    at BuiltinModule.compileForInternalLoader (node:internal/bootstrap/loaders:333:7)
    at requireBuiltin (node:internal/bootstrap/loaders:364:14)
    at node:internal/streams/operators:20:22

Node.js v20.0.0-pre
```
Hencer used the brand check convention also it seems to be the preferred convention across the webstreams code